src/source/reporting-query-and-response.rst

```diff
@@ -52,12 +43,6 @@ Parameter            Description
 ``searchPhrase``    String to match against an ad title
 ==================  ======================================
 
-Response
-************************************
-
-The response body of the API request is a JSON object of `Metrics Response`_ type. Here is a sample response for the sample request above.
-
-.. literalinclude:: /examples/metrics-response-v1.json
```

Comment:
same, why remove it?

Reply:
Added version agnostic exaples. Still think that it is an overkill cause we have examples listed bellow.